book_source/workflow/met_processing.RMD

```diff
@@ -0,0 +1,26 @@
+## Meteorological Data
```

Comment:
In the next PR, there needs to be better organization -- right now met and other input processing info is spread out over too many places in a way that's confusing and lacks sufficient cross references.